src/ansys/units/quantity.py

```diff
@@ -221,6 +222,36 @@ def to(self, to_units: Union[ansunits.Unit, str]) -> "Quantity":
 
         return Quantity(value=new_value, units=to_units)
 
+    def _relative_unit_check(self, __value, op: operator = operator.add) -> Quantity:
+        """
+        Checks relative units for temperature differences.
+
+        Parameters
+        ----------
+        __value : float, int, Quantity
+            The value to be added or subtracted to self.
+        op : operator, optional
+            The operation being performed. Default is addition.
+
+        Returns
+        -------
+        Quantity
+            Quantity instance changed to or from relative units.
+        """
+        if not isinstance(__value, ansunits.Quantity):
+            __value = ansunits.Quantity(__value)
+        # Adds or subtracts the units. Will be self.units unless temperatures
+        # are involved.
+        new_units = op(self._unit, __value._unit) or self.units
+        if __value.units != self.units:
+            new_prefix = "delta_" if __value.units.name.startswith("delta_") else ""
+            new_unit = self.units.name.removeprefix("delta_")
```

Comment:
```suggestion
           # TODO: issue 197 covers the fact that we are twiddling with the 
           # internals of unit representations here instead of using units in a more
           # encapsulated way
            new_prefix = "delta_" if __value.units.name.startswith("delta_") else ""
            new_unit = self.units.name.removeprefix("delta_")
```